src/System.Net.WebHeaderCollection/src/Resources/Strings.resx

```diff
@@ -138,13 +135,4 @@
   <data name="net_WebHeaderInvalidNonAsciiChars" xml:space="preserve">
     <value>Specified value has invalid non-ASCII characters.</value>
   </data>
-  <data name="net_WebHeaderMissingColon" xml:space="preserve">
-    <value>Specified value does not have a ':' separator.</value>
-  </data>
-  <data name="net_headerrestrict" xml:space="preserve">
-    <value>The '{0}' header must be modified using the appropriate property or method.</value>
-  </data>
-  <data name="net_emptystringcall" xml:space="preserve">
-    <value>The parameter '{0}' cannot be an empty string.</value>
-  </data>
 </root>
```

Comment:
Sure, I'd be happy to add the new line at the end while making changes here.

Though, FYI, I just left for vacation and won't be back until 4/3, so likely won't get to addressing any feedback until then.